languages/clojure/_sidebar.md

```diff
@@ -0,0 +1,11 @@
+[Start here](/)
```

Comment:
is this set as link necessary? would be confusing as it brings out 404 if clicked. I see c# is the same, might be reserved for later targeting?